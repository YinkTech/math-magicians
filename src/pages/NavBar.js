import React from 'react';
import { Link } from 'react-router-dom';

const NavBar = () => {
  return (
    <div className='nav'>
        <Link to="/"> Math Magicians </Link>
        <div className='links'>
        <Link to="/"> Home </Link>
          <Link to="/Calculator">Calculator </Link>
          <Link to="/Quotes">Quotes</Link>
        </div>
    </div>
  );
};

export default NavBar;