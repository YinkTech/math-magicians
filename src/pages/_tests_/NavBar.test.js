import react from "react";
import renderer from "react-test-renderer";
import NavBar from "./../NavBar";
import { BrowserRouter as Router } from "react-router-dom";

it("if it renders correctly", () => {
    const navbar = renderer
    .create(
        <Router>
            <NavBar />
        </Router>
    )
    .toJSON();
  expect(navbar).toMatchSnapshot();
});