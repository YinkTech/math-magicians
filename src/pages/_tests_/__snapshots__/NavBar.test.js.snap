// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`if it renders correctly 1`] = `
<div
  className="nav"
>
  <a
    href="/"
    onClick={[Function]}
  >
     Math Magicians 
  </a>
  <div
    className="links"
  >
    <a
      href="/"
      onClick={[Function]}
    >
       Home 
    </a>
    <a
      href="/Calculator"
      onClick={[Function]}
    >
      Calculator 
    </a>
    <a
      href="/Quotes"
      onClick={[Function]}
    >
      Quotes
    </a>
  </div>
</div>
`;
